lib/krikri/random_record_generator.rb

```diff
@@ -1,19 +1,36 @@
 module Krikri
   # Gets random records from the search index
   class RandomRecordGenerator
+    include Krikri::QaProviderFilter
 
     def initialize
-      @solr_repo = Blacklight::SolrRepository.new(Blacklight::Configuration.new)
+      @index_querier = Krikri::IndexQuerier.new
+      @default_query_params = { :id => '*:*',
```

Comment:
(I should have commented on the next line down.  It looks like it's pulling all documents, sorting by a random offset, and then returning the first row.  But I could be wrong, of course!)